components/common-data-items.js

```diff
@@ -602,6 +602,75 @@ FileDataItems.commonProperties = [
   "validation_error_detail",
 ];
 
+/**
+ * Display data items common to all FileSet objects.
+ */
+export function FileSetDataItems({ item, children }) {
+  return (
+    <>
+      {children}
+      {item.description && (
+        <>
+          <DataItemLabel>Description</DataItemLabel>
+          <DataItemValue>{item.description}</DataItemValue>
+        </>
+      )}
+      {item.aliases?.length > 0 && (
+        <>
+          <DataItemLabel>Aliases</DataItemLabel>
+          <DataItemValue>
+            <AliasList aliases={item.aliases} />
+          </DataItemValue>
+        </>
+      )}
+      {item.url && (
+        <>
+          <DataItemLabel>URL</DataItemLabel>
+          <DataItemValueUrl>
+            <a href={item.url} target="_blank" rel="noopener noreferrer">
+              {item.url}
+            </a>
+          </DataItemValueUrl>
+        </>
+      )}
+      {item.submitter_comment && (
+        <>
+          <DataItemLabel>Submitter Comment</DataItemLabel>
+          <DataItemValue>{item.submitter_comment}</DataItemValue>
+        </>
+      )}
+      {item.revoke_detail && (
+        <>
+          <DataItemLabel>Revoke Detail</DataItemLabel>
+          <DataItemValue>{item.revoke_detail}</DataItemValue>
+        </>
+      )}
+      {item.dbxrefs?.length > 0 && (
+        <>
+          <DataItemLabel>External Resources</DataItemLabel>
+          <DataItemValue>
+            <DbxrefList dbxrefs={item.dbxrefs} />
+          </DataItemValue>
+        </>
+      )}
+    </>
+  );
+}
+
+FileSetDataItems.propTypes = {
+  // file object common for all file types
```

Comment:
Really “FileSet object common for all FileSet types.”

Reply:
done